backend/src/main/java/saci/domain/service/LevelService.java

```diff
@@ -0,0 +1,46 @@
+package saci.domain.service;
+
+import java.util.List;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import saci.domain.model.Level;
+import saci.domain.service.exceptions.AlreadyExistsException;
+import saci.domain.service.exceptions.NotFoundException;
+import saci.infrastructure.LevelRepository;
+
+@Service
+@RequiredArgsConstructor
+public class LevelService {
+
+    private final LevelRepository LevelRepository;
```

Comment:
```suggestion
    private final LevelRepository levelRepository;
```

Variables should be camelCased. Just classes are CapitalCased

Reply:
on it